examples/jsm/controls/OrbitControls.js

```diff
@@ -1016,9 +1015,6 @@ class OrbitControls extends EventDispatcher {
 
 				scope.domElement.setPointerCapture( event.pointerId );
 
-				scope.domElement.addEventListener( 'pointermove', onPointerMove );
-				scope.domElement.addEventListener( 'pointerup', onPointerUp );
```

Comment:
I moved these listeners from `onPointerDown` to the constructor, where the other listeners are added. Since no `pointerdown` is fired when this bug occurs we need these listeners to be always active.

To compensate for this change I guarded both `onPointerMove` and `onPointerUp` with `isTrackingPointer`.